cli/run.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -e
 
-python -m cli.text2embeddings --s3 --device=cpu "${EMBEDDINGS_INPUT_PREFIX}" "${INDEXER_INPUT_PREFIX}"
\ No newline at end of file
+python -m cli.index_data --s3 "${INDEXER_INPUT_PREFIX}"
```

Comment:
Do we even need a `run.sh` here anymore?

Reply:
probably not, but that can be done in a follow-up I reckon